sdk/src/logs/async_batch_log_processor.cc

```diff
@@ -378,13 +161,7 @@ bool AsyncBatchLogProcessor::Shutdown(std::chrono::microseconds timeout) noexcep
   return true;
 }
 
-AsyncBatchLogProcessor::~AsyncBatchLogProcessor()
-{
-  if (synchronization_data_->is_shutdown.load() == false)
-  {
-    Shutdown();
-  }
-}
+AsyncBatchLogProcessor::~AsyncBatchLogProcessor() {}
```

Comment:
We should shutdown here.If it's shutdown in destructor of base class. the `AsyncBatchLogProcessor::Shutdown` will not be called.

Reply:
Agreed